allauth/account/apps.py

```diff
@@ -15,3 +15,7 @@ def ready(self):
             raise ImproperlyConfigured(
                 f"{required_mw} must be added to settings.MIDDLEWARE"
             )
+        from .receivers import send_password_change_notification
+        from .signals import password_changed
+
+        password_changed.connect(send_password_change_notification)
```

Comment:
I would prefer not to use signals for this -- just call the `send_notification()` method where the action occurs.